fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/db2/Db2Adapter.java

```diff
@@ -456,12 +456,10 @@ public void runStatement(IDatabaseStatement stmt) {
 
             String qname = ((DropColumn) stmt).getSchemaName() + "." + ((DropColumn) stmt).getTableName();
 
-            Db2AdminCommand runstats = new Db2AdminCommand("RUNSTATS ON TABLE " + qname + " WITH DISTRIBUTION AND DETAILED INDEXES ALL");
-            super.runStatement(runstats);
-
             String reorgCommand = "REORG TABLE " + qname;
             super.runStatement(new Db2AdminCommand(reorgCommand));
 
+            Db2AdminCommand runstats = new Db2AdminCommand("RUNSTATS ON TABLE " + qname + " WITH DISTRIBUTION AND DETAILED INDEXES ALL");
```

Comment:
make a lot sense, as I remember, most of the alter table operations requires reorg before the table is really usable ... otherwise it will always show reorg pending error for sql ...